### PR TITLE
Add colors to some missing UUIDs

### PR DESCRIPTION
**Pull request by kaminek**
and fix a empty display

### CODE REVIEW

File: internal/commands/ipaddress/show.go [hunk 1]
```diff
@@ -47,7 +47,11 @@ func (s *showCommand) Execute(exec commands.Executor, arg string) (output.Output
 					{Title: "Family:", Key: "access", Value: ipAddress.Family},
 					{Title: "Part of Plan:", Key: "access", Value: ipAddress.PartOfPlan, Format: output.BoolFormat},
 					{Title: "PTR Record:", Key: "access", Value: ipAddress.PTRRecord},
-					{Title: "Server UUID:", Key: "access", Value: ipAddress.ServerUUID},
+					{
+						Title: "Server UUID:",
+						Key:   "access", Value: ipAddress.ServerUUID,
```
Comment (moitias) [inline internal/commands/ipaddress/show.go:52]: The keys get used in json/yaml output so probably dont want to name them all the same... 

(sidenote: we should go through all of these i guess?)

Comment (moitias) [reply internal/commands/ipaddress/show.go:52]: (or; should we validate this even when rendering?)

Comment (kaminek) [reply internal/commands/ipaddress/show.go:52]: :scream: didn't even noticed it, yes it should be renamed properly

**Comment by moitias** [comment]
Besides the keys (which, I guess, could be another MR?), LGTM :+1:

**Pull request merged**